### ISSUE TITLE
Many false positives with 942360

### ISSUE THREAD

**Issue by Taiki-San** [coreruleset/coreruleset#1816]
### Description

The rule 942360 triggers many false positives.
Here are some of my samples:
- ` delete`
- `5desc`
- ` select`
- `34-delete`
- ` update`
- `/select`
- `(select`
- `Update: After...`
- `"desc"`
- `#Create System  `
- `/load.php`
- `/update-assets`
- `bla blabla live update chart`
- `.select-gws-banana`

I'm opening a PR expending the patches from @franbuehler (https://github.com/coreruleset/coreruleset/issues/1675) to the other patterns.

### Confirmation

[X] I have removed any personal data (email addresses, IP addresses,
    passwords, domain names) from any logs posted.

**Comment by dune73** [comment]
Sorry for the inconvenience and thank you for reporting.

I have tried a few of your payloads as follows:

```
$> curl localhost -d "x=update"
```

Some of them do trigger a false positive. My example does not.

Could you please check your payloads and if necessary also provide a PoC?

**Comment by Taiki-San** [comment]
We're not running those patterns directly on modsecurity, but I'll do my best.
In this case, I made a mistake extracting the data, there was a prepending space. Once added, it matches one of the patterns: https://regex101.com/r/BLWnxO/1

**Comment by Taiki-San** [comment]
Looking deeper at the regexes and the false positives, a subset are due to the use of `\W` matching a lot of characters not really relevant to SQL. I'm thinking of replacing it by a list of more relevant.
`as` experience a similar issue, although I suspect the pattern is broken (match `[\d\W]` before `AS` but shouldn't it be matching `[\w]`?)

My main concern with such a change is that I have limited visibility on what false negatives this would trigger.

**Comment by dune73** [comment]
Adding this to the issue chat tonight.

**Comment by Taiki-San** [comment]
Fixed in https://github.com/coreruleset/coreruleset/pull/1817
